inc/datahandlers/post.php

```diff
@@ -1607,11 +1607,15 @@ function insert_thread()
 
 				// Queue up any forum subscription notices to users who are subscribed to this forum.
 				$excerpt = my_substr($thread['message'], 0, $mybb->settings['subscribeexcerpt']).$lang->emailbit_viewthread;
```

Comment:
Isn't this line now useless as it is done below?

Reply:
OK, you are right. Everything else should be fine.